lib/cisco_node_utils/interface.rb

```diff
@@ -84,19 +84,21 @@ def channel_group
 
     def channel_group=(val)
       fail "channel_group is not supported on #{name}" unless @name[/Ethernet/i]
```

Comment:
It probably works as is since this file has an `attr :name` defined I agree it should be changed.